k9mail-library/src/main/java/com/fsck/k9/mail/internet/MimeMessageHelper.java

```diff
@@ -36,6 +36,10 @@ public static void setBody(Part part, Body body) throws MessagingException {
                 if (name != null) {
                     contentType += String.format(";\r\n name=\"%s\"", name);
                 }
+                boolean isFlowed = MimeUtility.isFormatFlowed(part.getContentType());
```

Comment:
We don't support creating `format=flowed` messages. It looks like this is only for the benefit of the new test in `MessageViewInfoExtractorTest`. I'd rather have the test set the header after adding the body than adding to this already horrible method.